ebu_tt_live/node/delay.py

```diff
@@ -42,7 +43,24 @@ def process_document(self, document, **kwargs):
                 # change the sequence identifier
                 document.sequence_identifier = self._document_sequence
 
-                # TODO: add an ebuttm:appliedProcessing element to the document metadata
+                if document.binding.head.metadata is None:
+                    document.binding.head.metadata = metadata.headMetadata_type(
+                        metadata.documentMetadata()
+                    )
+
+                if document.binding.head.metadata.documentMetadata is None:
+                    document.binding.head.metadata.documentMetadata = metadata.documentMetadata()
+
+                if len(document.binding.head.metadata.documentMetadata.appliedProcessing) == 0:
```

Comment:
It turns out you don't need to check for none existing already - I tried adding multiple and it worked! I'll push the change...